tests/test.web-ext.js

```diff
@@ -17,15 +17,14 @@ describe('web-ext', () => {
   // This is a smoke test just to make sure the command line script
   // doesn't explode.
 
-  it('prints the current version number', (done) => {
-    let pkg = JSON.parse(
-      readFileSync(path.join(__dirname, '..', '..', 'package.json')));
-    let cmd = 'bin/web-ext --version';
-    shell.exec(cmd, {silent: true}, (code, output) => {
-      assert.equal(output.trim(), pkg.version);
-      assert.equal(code, 0, output);
-      done();
-    });
+  it('shows help', (done) => {
+    let webExt = path.join(path.resolve(__dirname), '..', 'bin', 'web-ext');
+    shell.exec(`${webExt} --help`, {silent: true},
```

Comment:
If we don't need its contribute on the testing code coverage, how about opt to test it outside of the mocha tests, as a shell command? 

e.g. by adding to `.travis.yml` something like:

```
script: COVERAGE=y npm test && (cd ..; $TRAVIS_BUILD_DIR/bin/web-ext --help | grep -G "Usage: .*web-ext")
```